sunpy/map/mapsequence.py

```diff
@@ -274,9 +274,17 @@ def plot(self, axes=None, resample=None, annotate=True,
             Animation interval in ms
 
         plot_function : function
-            A function to be called as each map is plotted. Any variables
-            returned from the function will have their ``remove()`` method called
-            at the start of the next frame so that they are removed from the plot.
+            A function to be called as each map is plotted.
+            For more information see `sunpy.visualization.MapSequenceAnimator`.
+
+        Returns
+        -------
+        `matplotlib.animation.FuncAnimation`
+            A FuncAnimation instance.
```

Comment:
This line can be removed, as it's clear what the return type is from the above line.

Reply:
This is true but I kind of like the return section.